internal/helper/loadbalancer.go

```diff
@@ -164,8 +164,8 @@ func ReadCurrentRevisionFromLB(lb *yawolv1beta1.LoadBalancer) (int, error) {
 
 func GetNextRevisionForLoadBalancer(setList *yawolv1beta1.LoadBalancerSetList) (int, error) {
 	var highestRevision int
-	for _, set := range setList.Items {
-		rev, err := ReadRevisionFromLBS(&set)
+	for i := range setList.Items {
+		rev, err := ReadRevisionFromLBS(&setList.Items[i])
```

Comment:
Can we keep using the loop variable but disable that stupid linter?

Reply:
We could, but I think it is not that hard to read. There are 2 linters that did not like this code: 
```
               +lint *failed* | internal/helper/loadbalancer.go:167:2: rangeValCopy: each iteration copies 664 bytes (consider pointers or indexing) (gocritic)
               +lint *failed* |         for _, set := range setList.Items {
               +lint *failed* |         ^
               +lint *failed* | internal/helper/loadbalancer.go:168:35: G601: Implicit memory aliasing in for loop. (gosec)
               +lint *failed* |                 rev, err := ReadRevisionFromLBS(&set)
               +lint *failed* |                                                 ^

```